### ISSUE TITLE
[BUG]  v0.0.1.20210615.1 broke things, investigating

### ISSUE THREAD

**Issue by ggirelli** [ggirelli/radiantkit#178]
Unloading/loading not working anymore.

**Comment by ggirelli** [comment]
```bash
Traceback (most recent call last):
  File "/home/gire/.cache/pypoetry/virtualenvs/radiantkit-9dfFgQ5a-py3.8/bin/radiant", line 5, in <module>
    main()
  File "/home/gire/ownCloud/BiCro/Code/repos/radiantkit/radiantkit/scripts/radiant.py", line 67, in main
    args.run(args)
  File "/home/gire/ownCloud/BiCro/Code/repos/radiantkit/radiantkit/scripts/select_nuclei.py", line 324, in run
    images_list.extract_particles(Nucleus, [args.ref_channel], args.threads)
  File "/home/gire/ownCloud/BiCro/Code/repos/radiantkit/radiantkit/series.py", line 341, in extract_particles
    series.init_particles(particleClass, channel_list)
  File "/home/gire/ownCloud/BiCro/Code/repos/radiantkit/radiantkit/series.py", line 108, in init_particles
    self.__init_particles_intensity_features(channel_list)
  File "/home/gire/ownCloud/BiCro/Code/repos/radiantkit/radiantkit/series.py", line 73, in __init_particles_intensity_features
    pbody.init_intensity_features(self[name][1], name)
  File "/home/gire/ownCloud/BiCro/Code/repos/radiantkit/radiantkit/particle.py", line 129, in init_intensity_features
    pixels = self._region_of_interest.apply(img)[self.pixels]
  File "/home/gire/ownCloud/BiCro/Code/repos/radiantkit/radiantkit/selection.py", line 57, in apply
    return self.apply_to_pixels(img.pixels)
  File "/home/gire/ownCloud/BiCro/Code/repos/radiantkit/radiantkit/selection.py", line 53, in apply_to_pixels
    assert len(self._bounds) == len(pixels.shape), (self._bounds, pixels.shape)
AssertionError: ((slice(0, 103, None), slice(547, 718, None), slice(103, 202, None)), (0,))
```

**Comment by ggirelli** [comment]
Fixed in `0.0.1.20210616.1`.

**Comment by ggirelli** [comment]
Actually fixed in `v0.0.1.20210617.1`.
